### DISCUSSION TITLE
Mono 5.18 under Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by Yuri05** [mono/mono#21694] [Q&A]
Is it possible to install Mono **5.18** under Ubuntu **20.04**?
If yes: what are the steps?

Thank you.

**Comment by rochus-keller** [comment]
Have you tried to download the source from https://download.mono-project.com/sources/mono/mono-5.18.1.28.tar.bz2 and then just run "./configure" and then make from the source root directory?
